WikiWatershed/model-my-watershed
# Add P-factor to Other Model Data user configuration in gwlfe.

## Overview

This PR allows the user to configure a custom C-Factor and P-Factor for the Cropland LU/LC type in the gwlfe model.

Connects #3189 
Connects #3190

### Demo

![Dec-17-2019 14-19-03](https://user-images.githubusercontent.com/2320142/71027188-bd596a80-20d8-11ea-9fb5-b807e1d7db97.gif)

## Testing Instructions

* Run this branch
* Select a HUC 12
* Go to Model -> Watershed Multi-year Model
* Click "Add changes to this area"
* Click "Settings"
* Go to tab "Other Model Data"
* See that there is C-factor and P-factor for Cropland with the correct default value, change, save, and reopen to verify the change sticks.

## Checklist
